### DISCUSSION TITLE
Bed on thumbnail

### DISCUSSION THREAD

**Discussion by Gudzonskiy** [OrcaSlicer/OrcaSlicer#7545] [Ideas]
Please add the "bed on thumbnail" option like in superslicer.

**Comment by Nite16** [comment]
Agree that this would be a nice add.

**Comment by carlosjln** [comment]
If this ever happens, it must be an option instead of some hardcoded decision. Personally I don't like PrusaSlicer thumbnail precisely because it exports the build plate or the background grid which is something I like about Orca Slicer, the model thumbnail is clean on the printer screen. Less clutter is better.

**Reply by Nite16** [reply]
Agreed! Purely optional. I use it on Klipper/Mainsail where there's more resolution.
